charms/kfp-api/lib/charms/grafana_k8s/v0/grafana_dashboard.py

```diff
@@ -21,8 +21,9 @@
 use case of a charm which bundles dashboards and provides a
 `provides: grafana-dashboard` interface,
 
-    grafana-dashboard:
-      interface: grafana_dashboard
+    requires:
```

Comment:
should we update this and the prometheus_scrape lib files in the other repos? I haven't been doing this

Reply:
That was recommended by Observability Team. [Updated version](https://github.com/canonical/grafana-k8s-operator/blob/main/lib/charms/grafana_k8s/v0/grafana_dashboard.py) handles some exceptions I had in the previous one.